nmostesting/NMOSTesting.py

```diff
@@ -460,10 +460,10 @@ def run_tests(test, endpoints, test_selection=["all"]):
                 "selector": endpoints[index]["selector"],
                 "spec": None  # Used inside GenericTest
             }
+            apis[api_key]["spec_path"] = CONFIG.CACHE_PATH + '/' + spec_key

```

Comment:
This could go inside the dict construction?

Reply:
Good point. I'll update that